src/main/java/com/kenshoo/xlsx2csv/XlsxToCsvConverter.java

```diff
@@ -86,15 +89,24 @@ private boolean isLastLine(boolean hasNext) {
         return !hasNext;
     }
 
-    private Iterator<Row> getRowIterator(InputStream inputStream) {
+    private Optional<Iterator<Row>> getRowIterator(InputStream inputStream) {
         final Workbook workbook = StreamingReader.builder()
                 .rowCacheSize(10)
                 .bufferSize(1024)
```

Comment:
an additional constructor (with a builder) sounds like a great option - we'll probably add more such options as we go ahead